src/main/webapp/app/home/notification-set.component.js

```diff
@@ -20,23 +20,214 @@
         vm.range = range;
         vm.noOfPages = 1;
         vm.currentPage = 0;
-        vm.count=0;
+        vm.count = 0;
+        vm.reports;
 
         active();
 
-        function active(){
+        function active() {
             // vm.alerts=vm.releaseAlert.alerts;
             // vm.count=vm.releaseAlert.count;
             connectWebSocket();
             getScheduledReportsCount();
+            vm.reports=[
```

Comment:
are you testing with hard code content

Reply:
yes I have already added in description 
but now I am working with API data